torch/csrc/distributed/rpc/utils.h

```diff
@@ -39,6 +40,18 @@ TORCH_API std::pair<std::vector<char>, std::vector<at::Tensor>> wireDeserialize(
     const void* data,
     size_t data_size);
 
+// Tensorpipe message conversion
+struct TensorPipeEntry {
+  tensorpipe::Message message;
+  std::vector<torch::Tensor> reservedTensors;
+  std::vector<std::vector<uint8_t>> copiedTensors;
+};
+
+TORCH_API TensorPipeEntry tensorpipeSerialize(const Message& rpcMessage);
+
+TORCH_API Message
```

Comment:
Instead of putting this in utils, might be better to have something like torch/csrc/distributed/rpc/tensorpipe/serialize.{h, cpp}. The rest of the tensorpipe code can also go inside the tensorpipe/ directory.

Reply:
I'm not sure if it deserves a separate folder for tensorpipe. As these serialization and deserialization functions doing similar with wireSerialize, maybe let's keep in this file for now